pkg/eventprocessor/eventfeed/impl/eventfeed.go

```diff
@@ -184,6 +184,20 @@ func (ef *EventFeed) Start(
 					strings.Contains(err.Error(), "block range is too wide") {
 					ef.maxBlocksFetchSize = ef.maxBlocksFetchSize * 80 / 100
 				} else {
+					// If we get a "looksback" error it means that history is not available
+					// for this chain. It happens in Filecoin based chains, where the
+					// history is not available in non archive nodes.
+					// In this case, we just move the fromHeight
+					// to the current head minus 1995 blocks and ignore the past events.
+					// This is temporary until we have a better access to archive nodes.
+					if strings.Contains(err.Error(), "lookbacks of more than") {
```

Comment:
Ignoring events deeper than 1995 blocks because non archival nodes don't have access to it.